.github/workflows/android.yaml

```diff
@@ -29,6 +23,8 @@ jobs:
     with:
       platform: chromium_android-arm64
       nightly: ${{ github.event.inputs.nightly }}
+    secrets:
```

Comment:
Can this be done per worfklow, rather than per job ? I would think per workflow global settings are supported.

Reply:
I'm not sure. The docs on the subject are quite sparse, there's just a single paragraph and it shows this.

`env:` seems to be a special case and is supported on a workflow level, rather than job level. Not sure if the same is true for secrets.

Reply:
My reccomendation: Keep a small standalone repo where things like these can be tested quickly

Reply:
Tested in a tiny repo. Secrets can not be configured globally.

`The workflow is not valid. .github/workflows/linux.yaml (Line: 23, Col: 1): Unexpected value 'secrets'`